### DISCUSSION TITLE
How to update phoenixd

### DISCUSSION THREAD

**Discussion by danielcharrua** [ACINQ/phoenixd#108] [Q&A] [answered]
Is there any guide on how to update the software? I installed using wget as stated on the "[get started](https://phoenix.acinq.co/server/get-started)"

If a new version is released, I can use the same command to get the software and update it? I will loose any actual config or need to do something else?

**Comment by pm47** [comment, marked as answer]
Simply replace the binary and restart the daemon. Phoenixd inherits from the same simplicity as the mobile version, upgrades are fully transparent.
